llvm/include/llvm/Passes/PassBuilder.h

```diff
@@ -478,7 +478,8 @@ class PassBuilder {
   /// This extension point allows adding optimization right after passes that do
   /// basic simplification of the input IR.
   void registerPipelineEarlySimplificationEPCallback(
-      const std::function<void(ModulePassManager &, OptimizationLevel)> &C) {
+      const std::function<void(ModulePassManager &, OptimizationLevel, bool)>
```

Comment:
Can we pass the full LTOPhase instead?

Reply:
Yes of course. I was in the same shoes and not sure which one is better, since the pipeline is only used in two places.

Reply:
I think using the full phase is more future-proof, in case someone wants to distinguish thin/full LTO in the future.